data/ecosystems/s/saucerswaplabs.toml

```diff
@@ -0,0 +1,22 @@
+# Ecosystem Level Information
+title = "SaucerSwap Labs"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/saucerswaplabs]
```

Comment:
This is invalid toml, but overall the content looks good. Please fix and we can get it merged.

Reply:
Thanks @jubos — apologies on the mistypes there; I've gone ahead and updated!